### PR TITLE
24 question

### PR DESCRIPTION
**Pull request by AlaaTaima**
Relates #24 
Questions-page :
The user can write his story by answering three questions at this page, and also  the user has a choice to answer or skip the question. every question consist of two parts , the title and explanation so, the user will find a good space to talk about his feelings.

### CODE REVIEW

**Comment by Fatmasiam** [comment]
Write description

File: src/containers/About/about.css [hunk 1]
```diff
@@ -28,3 +28,32 @@
     align-items: center;
     justify-content: space-around
 }
+
+.about__img {
+    width: 30vh;
```
Comment (yosefanajjar) [inline src/containers/About/about.css:33]: why width with vh unit?

File: src/containers/About/about.css [hunk 2]
```diff
@@ -28,3 +28,32 @@
...
+    .about__heading .main-head {
+        font-size: 45px;
+    }
+    .about__title {
+        font-size: 38px;
+    }
+    .about__content {
+        font-size: 30px;
+    }
+}
```
Comment (yosefanajjar) [inline src/containers/About/about.css:59]: add new line

File: src/containers/Questions/questions.jsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+  },
+  {
+    id: 3,
+    heading: 'Iam looking to develop',
+    percent: 100,
+  },
+];
+
+class Questions extends React.Component {
+  constructor(props) {
```
Comment (yosefanajjar) [inline src/containers/Questions/questions.jsx:28]: We are using es7 in our code style and we don't have constructors in this style

File: src/containers/Questions/questions.jsx [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    message.success("You don't make an entry today");
+    const { history } = this.props;
+    history.push('/home');
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value });
+  };
+
+  next() {
```
Comment (yosefanajjar) [inline src/containers/Questions/questions.jsx:48]: why are you using es5 and in the previous function your wrote it in es6?

File: src/routes/App/style.css [hunk 5]
```diff
@@ -24,4 +24,4 @@
 
 .App-link {
   color: #09d3ac;
-}
+}
```
Comment (yosefanajjar) [inline src/routes/App/style.css:27]: add new line

File: src/containers/Questions/questions.jsx [hunk 6]
```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
+import schema from './questionValidation';
+import './questions.css';
+
+const { TextArea } = Input;
+
+const entryData = [
```
Comment (yosefanajjar) [inline src/containers/Questions/questions.jsx:9]: I think the file would be more readable if the data was in another file, just a suggestion :)

File: src/containers/Questions/questions.jsx [hunk 7]
```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { Button, message, Input, Progress, Icon, Popconfirm } from 'antd';
```
Comment (yosefanajjar) [inline src/containers/Questions/questions.jsx:3]: add a line break after the first party and third party modules

File: src/containers/Questions/questions.jsx [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+    this.state = {
+      current: 0,
+      title: '',
+      content: '',
+      errors: {},
+    };
+  }
+
+  confirm = e => {
+    message.success("You don't make an entry today");
```
Comment (yosefanajjar) [inline src/containers/Questions/questions.jsx:39]:
```suggestion
    message.success("You didn't add an entry today :(");
```

File: src/containers/Questions/questions.jsx (hunk 3 above)
Comment (AlaaTaima) [reply src/containers/Questions/questions.jsx:28]: I know , but the rules for unused constructor haven't merged yet , so Iam waiting to overcome conflict.

**Comment by yosefanajjar** [comment]
We agreed to use async await why are you using promisess?

**Comment by AlaaTaima** [comment]
> We agreed to use async await why are you using promisess

**Pull request closed without merging**